index.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Alick's Portfolio</title>
+  <meta name="Alick Nyirenda" content="Alick Portfolio">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <header>
+    <p class="header-text">Alick Nyirenda </p>
+    <a href="#"><i class="header-icon fa fa-bars fa-lg"></i></a>
+  </header>
+  <main>
+  <section class="headline">
+    <h1 class="headline-text-1">Alick Nyirenda</h1>
+    <p class="headline-text-2">Software Developer</p>
+  </section>
```

Comment:
- [ ] This section is missing the proper background image. You must stick to the design as much as possible (e.g. font, colours, images, margins) using the templates in Figma.
_See images for better comparison._
**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/121370380-8b058380-c8f1-11eb-9dbc-a4a891cf1ed4.png) | ![image](https://user-images.githubusercontent.com/49621279/121370467-9ce72680-c8f1-11eb-9886-d93cccd14e4a.png)